### PR TITLE
Don't overwrite existing configuration

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: app/models/riiif/image.rb [hunk 1]
```diff
@@ -5,9 +5,9 @@ class Image
     extend Deprecation
 
     class_attribute :file_resolver, :info_service, :authorization_service, :cache
-    self.file_resolver = FileSystemFileResolver.new(base_path: ::File.join(Rails.root, 'tmp'))
-    self.authorization_service = NilAuthorizationService
-    self.cache = Rails.cache
+    self.file_resolver ||= FileSystemFileResolver.new(base_path: ::File.join(Rails.root, 'tmp'))
```
Comment (jcoyne) [inline app/models/riiif/image.rb:8]: How could it overwrite if the attribute was just declared on line 7?

Comment (cbeer) [reply app/models/riiif/image.rb:8]: I'm seeing weird behavior within a Rails app, probably something to do with auto-reloading, where my file resolver (configured in an initializer) gets wiped out and replaced with this default.

Comment (jcoyne) [reply app/models/riiif/image.rb:8]: Can you add a comment about that so that your changes don't get inadvertently removed?

Comment (mcritchlow) [reply app/models/riiif/image.rb:8]: fwiw, I'm seeing the same running a Spotlight app. in our initializer for riiif, [we set the `file_resolver` to CarrierWave](https://gitlab.com/surfliner/surfliner/-/blob/master/starlight/config/initializers/riiif.rb#L9)

However, if i check (via `binding.pry`) during a csv upload, it's been flipped to the default:
```
[2] pry(#<Spotlight::AddUploadsFromCSV>)> Riiif::Image.file_resolver
=> #<Riiif::FileSystemFileResolver:0x00005612c3ff7810 @base_path="/home/starlight/app/tmp">
```

**Comment by dunn** [comment]
is there anything still holding this up?

File: app/models/riiif/image.rb (hunk 1 above)
Comment (dunn) [reply app/models/riiif/image.rb:8]: same as @mcritchlow, even with this patch applied

**Comment by jcoyne** [comment]
Yes, the requested comment.

File: app/models/riiif/image.rb (hunk 1 above)
Comment (fabianzwodrei) [reply app/models/riiif/image.rb:8]: same here. using rails 6 - worked in rails 5. config from initializer get overriden

**Pull request closed without merging**